11-node-frameworks/Egor_Emelyanov/index.js

```diff
@@ -0,0 +1,56 @@
+const Koa = require('koa');
+const router = require('koa-router');
+const serve = require('koa-static');
+const session = require('koa-generic-session');
+const bodyParser = require('koa-bodyparser');
+const joi = require('joi');
+
+const app = new Koa();
+const _ = router();
+app.keys = ['kek', 'lol'];
+
+let formData = {};
+
+const schema = joi.object({
+  name: joi.string().required(),
+  description: joi.string().min(3),
+});
+
+app.use(serve('assets/text'));
+app.use(serve('assets/alohadance'));
+app.use(session());
+app.use(bodyParser());
+
+function get() {
```

Comment:
keep in mind that counter should work for any request to the server, even posting form. Split counting and getting the "session counter"

Reply:
basically it should be middleware which works for each request to the server, without any response